exercises/tom_clayton/ex02_oo_basics/src/Triangle.h

```diff
@@ -0,0 +1,31 @@
+#ifndef TRIANGLE_H_
+#define TRIANGLE_H_
+
+#include "Shape.h"
+
+class Triangle : public Shape
+{
+  private: 
+    double m_height;
+    double m_base;
+
+  public:
+    Triangle(double height, double base);
+
+    void setBase(double base);
+
+    void setHeight(double height);
+
+    double calcPerimeter(double height, double base);

```

Comment:
As this method is overriding a virtual function in `Shape` you can use the [override](https://en.cppreference.com/w/cpp/language/override) keyword which will check if it matches the function it is overriding when compiling